### ISSUE TITLE
Can't build version 0.0.30 (macOS)

### ISSUE THREAD

**Issue by Itheme** [stellar/bridge-server#97]
Checked out at commit 7a0a8f0444e7e83d67251aadb635ad3a64cb9f32 (0.0.30)
`gb build` says

> fatal: A branch named 'build' already exists.

go version go1.4-bootstrap-20170531 darwin/amd64
macOS 10.12

Also`godoc -goroot=. -http=:6060` doesn't show any documents for me
Webpage says
> open src/github.com/stellar/gateway: no such file or directory

**Comment by bartekn** [comment]
You have a naming conflict (multiple `gb` binaries). Try `$GOPATH/bin/gb build`.

**Comment by Itheme** [comment]
Thank you @bartekn. After using `$GOPATH/bin/gb build` it did build on a Mac.

Additional details for those who might google this problem later: It happened to me on a clean machine with no prior go (and gb) installation. Same behaviour for an Ubuntu 16 machine with golang 1.6.2

After installing go in both cases, I've added `export GOPATH=$HOME/gocode` into .zshrc, installed gb via `go get github.com/constabulary/gb/...` and ran `gb build`. First it silently fails, then I've got this error.

Ubuntu build failed with `app.go:239: undefined: http.StatusPermanentRedirect`, but I'll investigate this one later
